challenges/logic/todo/luka_jelacic/lib/blocs/todo_state.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:luka_jelacic/todoItem.dart';
+
+enum TodoStateStatus {
+  InitialState,
```

Comment:
Mala slova
 https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names